app/views/play/modal/PlayHeroesModal.coffee

```diff
@@ -313,7 +314,9 @@ module.exports = class PlayHeroesModal extends ModalView
 
   askToBuyGemsOrSubscribe: (unlockButton) ->
     @$el.find('.unlock-button').popover 'destroy'
-    if me.canBuyGems()
+    if me.isStudent()
+      popoverTemplate = earnGemsPromptTemplate {}
```

Comment:
should be controlled by classroom items feature flag?

Reply:
PlayHeroes modal does not show up at all for a student if the classroomItems feature flag is off. So, it is not required to do the check here. If the modal shows up it means these features are valid.

Reply:
Play Heroes modal does not get displayed anywhere if the classroom items are turned OFF. If this method is called, it means that the modal is displayed and the feature flag is ON. Not necessary to check for it again.

Reply:
Another note: We are not defining `popoverTemplate` outside of the `if` statements. This is working because CoffeeScript variable scoping is strange.